embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -39,6 +40,27 @@ internal sealed class SchemaType(
         override val attrs = mapOf("view.name" to viewName)
     }
 
+    /**
+     * Represents a tap breadcrumb event.
+     * @param viewName The name of the view that the tap event occurred in.
+     * @param type The type of tap event. "tap"/"long_press". "tap" is the default.
+     * @param coords The coordinates of the tap event.
+     */
+    internal class TapBreadcrumb(
+        viewName: String?,
+        type: String = "tap",
+        coords: String
+    ) : SchemaType(
+        EmbType.Ux.Tap,
+        TAP_BREADCRUMB
+    ) {
+        override val attrs = mapOf(
+            "view.name" to viewName,
+            "tap.type" to type,
+            "tap.coords" to coords
```

Comment:
Not for the migration now, but something to think about for the future when we want to make this a semantic convention for all of OTel - do we want to have a "unit" field so tap coordinates can be in % of the viewport, device pixels, device independent pixels, etc?